scripts/iso_name_tagging.py

```diff
@@ -37,9 +37,9 @@
     command:
         iso_name_tagging.py -b BED -s SAM
     new name:
-        hsa-miR-524-5p|0|0|11M3I11M|22
+        ""
```

Comment:
Why did this change? Would be good to give an explanation for each example. It's not clear why no tag is added here.

Also, are you sure that an empty string doesn't cause problems? Does it not violate the SAM specs?

Reply:
After looking at the documentation and some related issues on GitHub, I found [this answer](https://github.com/GFA-spec/GFA-spec/issues/124#issuecomment-2174511060) which I believe explains quite well  why empty strings are allowed on the optional string type tag Z (the one used in this script)